rest_api/openapi.yaml

```diff
@@ -12,12 +12,12 @@ securityDefinitions:
     in: header
     name: Authorization
 paths:
-  /authorization:
+  /authentication:
```

Comment:
Just wondering if this makes more sense as a verb:  `authenticate` vs `authentication`